src/rail/estimation/algos/deepdisc.py

```diff
@@ -23,37 +23,33 @@
 from deepdisc.training.trainers import (return_evallosshook,
                                         return_lazy_trainer, return_optimizer,
                                         return_savehook, return_schedulerhook)
-from detectron2.config import LazyConfig, get_cfg, instantiate
+from detectron2.config import LazyConfig, get_cfg, instantiate, CfgNode
 from detectron2.engine import launch
 from detectron2.engine.defaults import create_ddp_model
 from rail.core.common_params import SHARED_PARAMS
 from rail.core.data import Hdf5Handle, JsonHandle, QPHandle, TableHandle
 from rail.estimation.estimator import CatEstimator, CatInformer
 
-from rail.deepdisc.configs import *
-
 
 def train(config, all_metadata, train_head=True):
+    
     cfgfile = config["cfgfile"]
     batch_size = config["batch_size"]
-    numclasses = config["numclasses"]
     output_dir = config["output_dir"]
-    output_name = config["output_name"]
+    run_name = config["run_name"]
     print_frequency = config["print_frequency"]
-    epochs = config["epochs"]
-    head_iters = config["head_iters"]
-    full_iters = config["full_iters"]
+    epoch = config["epoch"]
+    head_epochs = config["head_epochs"]
+    full_epochs = config["full_epochs"]
     training_percent = config["training_percent"]
 
-    cfg = get_lazy_config(cfgfile, batch_size, numclasses)
-    cfg_loader = get_loader_config(output_dir, batch_size)
 
-    e1 = epochs * 15
-    e2 = epochs * 10
-    e3 = epochs * 20
-    efinal = epochs * 35
+    e1 = epoch * head_epochs
+    e2 = epoch * 10
+    e3 = epoch * 20
+    efinal = epoch * full_epochs
```

Comment:
minor point (not necessary to change in this PR) - do we really need to define these (`e1`, `e2`, `e3`, `efinal`)? Or can we use the `epoch * X` directly?